coral/src/app/features/topics/details/overview/TopicOverview.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  Box,
+  Button,
+  Card,
+  Grid,
+  GridItem,
+  Typography,
+} from "@aivenio/aquarium";
+import add from "@aivenio/aquarium/dist/src/icons/add";
+import { useMemo } from "react";
+import { Link } from "react-router-dom";
+import { useTopicDetails } from "src/app/features/topics/details/TopicDetails";
+import StatsDisplay from "src/app/features/topics/details/components/StatsDisplay";
+import { getTopicStats } from "src/app/features/topics/details/utils";
+
+function TopicOverview() {
+  const { topicName, environmentId, topicOverview, topicSchemas } =
+    useTopicDetails();
+
+  const stats = useMemo(() => getTopicStats(topicOverview), [topicOverview]);
```

Comment:
why are we using useMemo? 

(that's a: "huh I'm interested bc I didn't think of using it here, wanna learn!!" and not a "what are you doing 😾 " question 😂 )

Reply:
Because `getTopicStats` will spread and loop over all the subscriptions, and there can be up to 1500-2000 of them, which may make it a little bit of an expensive operation. Additionally, we don't need to re-compute these stats on every render, but only when the `topicOverview` data changes ^^

Reply:
💡  OK, thanks for the explanation! I need to keep useMemo in mind more, I usually don't remember it exists.